cpu/stm32/Makefile.features

```diff
@@ -34,6 +34,25 @@ ifneq (,$(filter $(STM32_WITH_BKPRAM),$(CPU_MODEL)))
   FEATURES_PROVIDED += backup_ram
 endif
 
+STM32_WITH_VBAT = stm32f031% stm32f038% stm32f042% stm32f048% \
+                  stm32f051% stm32f058% stm32f071% stm32f072% \
+                  stm32f078% stm32f091% stm32f098% \
+                  stm32f2% \
+                  stm32f3% \
+                  stm32f4% \
+                  stm32f7% \
+                  stm32g0% \
+                  stm32g4% stm32gbk1cb \
+                  stm32l4% \
+                  stm32l5% \
+                  stm32u5% \
+                  stm32wb% \
+                  stm32wl% \
+
+ifneq (,$(filter $(STM32_WITH_VBAT),$(CPU_MODEL)))
+  FEATURES_PROVIDED += vbat
```

Comment:
The name `periph_vbat` is surely comprehensible. But I personally would not call it like that because probably no datasheet would list it as an extra peripheral because it is a feature of the ADC. I saw that there is `cpu/efm32/drivers/coretemp/coretemp.c` which is also samples an ADC channel. So I decided to implement VBAT  similar to this. So you think it should rather be `periph_vbat`?

Reply:
Although that is `efm32_coretemp` (and I think a generic version should be `periph_coretemp`).

Generally I understand the `periph_` prefix as something that needs a per-CPU implementation as opposed to a generic module. (I see that this makes the case for `periph_backup_ram`…)

Reply:
Yes I think it should just be `coretemp` :sweat_smile:. Because if my use case or application requires the core temperature, I´d rather do `USEMODULE += coretemp` and not care if my CPU is `efm32`.  Or better, `coretem` could pull in `efm32_coretemp` for `efm32` but that´s not the case right now.

Reply:
Ah yes or `periph_coretemp` like you said.

Reply:
Anyway, I have no problem with `periph_`, if you think it´s better.